### PR TITLE
Update lynis

### PR DESCRIPTION
**Pull request by paolostivanin**
On 15-SP4, the test was failing because it found `ERROR|WEAK|UNSAFE` even though the baseline file had these items in it.
This happens when the `$baseline` and `$current` file have one or more differences, because in such case only the `$current` file is being checked and, since there are `ERROR|WEAK|UNSAFE` items, the test fails.
This PR add some exceptions when checking for `lynis_fail` based on what is already present in the baseline file.

VRs:
- 15-SP3: 
  - https://openqa.suse.de/tests/9085821
  - https://openqa.suse.de/tests/9085822
- 15-SP4:
  - https://openqa.suse.de/tests/9085820
  - https://openqa.suse.de/tests/9085809
  - https://openqa.suse.de/tests/9085810

### CODE REVIEW

**Comment by rfan1** [comment]
CI check failed.

**Comment by paolostivanin** [comment]
The CI fails, but I don't get why:

```
 ~/git_projs/os-autoinst-distri-opensuse [fix-lynis-data L|✔] 
08:26 $ ./tools/tidy --check
✔ ~/git_projs/os-autoinst-distri-opensuse [fix-lynis-data L|✔] 
08:26 $ 
```

File: lib/lynis/lynistest.pm [hunk 1]
```diff
@@ -205,6 +206,52 @@ sub parse_lynis_section_content {
     return @section_content;
 }
 
+sub check_exceptions {
```
Comment (rfan1) [inline lib/lynis/lynistest.pm:209]: LGTM, only minor comment here,
could you please add some description why so many 'exceptions' here?

**Review by rfan1 — APPROVED** [review]
LGTM

File: lib/lynis/lynistest.pm (hunk 1 above)
Comment (paolostivanin) [reply lib/lynis/lynistest.pm:209]: sure thing!

**Review by jouyingbin — COMMENTED** [review]
LGTM. :)

**Pull request merged**